api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -45,6 +47,19 @@ public void willStart() {
      * @see ModuleActivator#started()
      */
     public void started() {
+        String refAppLocationGlobalProperty = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME);
```

Comment:
What does ref app global property mean?

Reply:
better to change as "locationUserPropertyName". Do you agree?